src/main.js

```diff
@@ -127,3 +127,5 @@ app.on('web-contents-created', (e, contents) => {
     showInspectElement: false,
   });
 });
+
+app.allowRendererProcessReuse = false;
```

Comment:
what does this mean?

Reply:
This essentially enables older overrides that Electron had in place to ensure renderer processes are restarted on every navigation. Unfortunately, it is currently needed because of our dependency on [Node Serialport](https://github.com/serialport/node-serialport)

There's a good discussion [here](https://github.com/electron/electron/issues/18397) from the Electron team and a Serialport specific issue [here](https://github.com/serialport/node-serialport/issues/2051).

One important note is that we should be able to stop using Serialport entirely in favor of WebSerial/WebUSB given that Chromium [enabled those by default in v89](https://www.chromestatus.com/feature/6577673212002304) and the [current Electron release](https://www.electronjs.org/releases/stable?version=13) includes v91. Once we make those updates in CodeStudio, we can remove our dependency and this flag (our builds will also become a lot simpler as we will no longer have any native dependencies to rebuild).

I am going to add a comment in the code so we know when this is safe to remove.